java/pom.xml

```diff
@@ -33,7 +33,7 @@
     <dep.junit.jupiter.version>5.4.0</dep.junit.jupiter.version>
     <dep.slf4j.version>1.7.25</dep.slf4j.version>
     <dep.guava.version>20.0</dep.guava.version>
-    <dep.netty.version>4.1.48.Final</dep.netty.version>
+    <dep.netty.version>siren-4.1.48-1-SNAPSHOT</dep.netty.version>
```

Comment:
Netty version should be bumped to release `siren-4.1.48-1` before merging to master.

Reply:
Netty `siren-4.1.48-1` has been deployed, done https://github.com/sirensolutions/arrow/pull/16/commits/e7aa338dc348e189b998afef5e207aa801692d55